crates/ln-dlc-node/src/ln/app_event_handler.rs

```diff
@@ -172,13 +200,27 @@ where
                 user_channel_id,
                 ..
             } => {
-                common_handlers::handle_channel_ready(
-                    &self.node,
-                    &self.pending_intercepted_htlcs,
+                let user_channel_id = UserChannelId::from(user_channel_id).to_string();
+
+                tracing::info!(
                     user_channel_id,
-                    channel_id,
-                    counterparty_node_id,
-                )?;
+                    channel_id = %channel_id.to_hex(),
```

Comment:
I think it doesn't need `%` here because it's already a string (same the line below).